src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -39,21 +41,30 @@ function getPaypalConfig() {
       helpers.paymentFromComponent(state.data, component);
     },
     onCancel: (data, component) => {
+      window.paypalTerminatedEarly = false;
```

Comment:
why is this line needed in all the callbacks?
I guess it would be enough only once the properties are defined like the `showPayButton: true,`.